scripts/install.sh

```diff
@@ -52,22 +48,43 @@ if [ -n "$NEEDS" ]; then
     exit 1
 fi
 
-status "Downloading ollama..."
-curl --fail --show-error --location --progress-bar -o $TEMP_DIR/ollama "https://ollama.ai/download/ollama-linux-$ARCH"
+TEMP_DIR=$(mktemp -d)
+cleanup() {
```

Comment:
```suggestion
finish() {
```

This is doing more than cleanup now